README.md

```diff
@@ -10,7 +10,7 @@ A simple, responsive lightbox component for displaying an array of images.
 
 
 ```bash
-npm install --save react-images
+yarn add react-images --save
```

Comment:
`--save` switch does not exists with yarn. It adds it automatically to dependencies.

Reply:
changed